### DISCUSSION TITLE
simple project

### DISCUSSION THREAD

**Discussion by kmchen13** [Motion-Project/motion#1797] [Q&A]
Hi, I have a ZTE F6640 router and will install a wifi conected camera on its local network. I want to use motion on my laptop when I'm in travel to access the camera from the outside.
Is that possible without any other server on my local net, only a camera ?
I don't care if chinese want to see me naked on my terace :-)

**Comment by Mr-Dave** [comment]
Actually, it probably isn't the camera view that would inspire people.  It would be whether opening that camera view also exposes other things that bad actors could prey upon to get into your home network and exploit to do malicious things.

To answer the specific question.  Motion would be the software that would be running on a machine in the local network.  You'd set it up with TLS (https) along with a strong username/password, disable remote changes to parameters (webcontrol_params) and then forward the stream port via the router.  You'd then figure out your external IP address and whether it is dynamic (changes upon router reboot).  Motion would be running at your home and serving up a page that you could view via a web page pointed to your home's external IP address.  If the IP is dynamic, you'd need to get some service(possibly for a fee) that provides a consistent web address and your server updates that service with new IP addresses as they change.
